src/components/sideBar/menuItem/types.ts

```diff
@@ -11,9 +11,11 @@ export interface SubsectionMenuItemType {
   subsections?: SubsectionMenuItemType[];
   path?: string;
   label: string;
+  authorizedOnly: boolean;
```

Comment:
authenticatedOnly 

unless you want to pair this with Permissions

Reply:
mm no you are right, i will use string[] for section permission